src/views/nunjucks/abstraction-reform/edit-data.njk

```diff
@@ -39,5 +33,22 @@
   integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
   crossorigin="anonymous"></script>
 <script src="/public/javascripts/json-forms-toggle.js"></script>
-
+<script src="/public/javascripts/accessible-autocomplete.min.js"></script>
+<link href="/public/stylesheets/accessible-autocomplete.min.css" rel="stylesheet" />
+<script nonce={{nonces.script}}>
```

Comment:
If this inline code chunk is the same as the one in above, would it be worth moving it to its own JS file?